### ISSUE TITLE
.NET Conf Focus on IoT

### ISSUE THREAD

**Issue by pjgpetecodes** [dotnet-foundation/events#6]
There's a burgeoning community and a mass of content around Microsoft IoT.

It would be amazing to leverage this, and the support in the community, to put on a fab event around all of the tools, services and applications around IoT.

It would also be a great way for the community to meet the various awesome members of the Microsoft IoT Team, both in Advocacy / DevRel and Engineering. 

Likewise, the community can help the exposure for the amazing work the Microsoft IoT Team are doing to create a fantastic ecosystem, by delivering content too.

Of course, as IoT touches so many facets of Azure, there would be opportunity to bring in folks from other departments too.

Thanks!

**Comment by shaggygi** [comment]
Of course, this event doesn't need to be strictly focused around Azure IoT.  It should also include the good work done over in https://github.com/dotnet/iot, as well.

**Comment by josesimoes** [comment]
Terrific idea! And let's bring in also [.NET nanoFramework](https://github.com/nanoframework) 😉 
Work has been done to align APIs with Core IoT, mirroring the Device Bindings, etc...

**Comment by nelsonmorais** [comment]
And what about expanding it also to IIoT?

**Comment by pjgpetecodes** [comment]
> And what about expanding it also to IIoT?

Agreed... And AIoT

**Comment by SteveDesmond-ca** [comment]
I'm always happy to talk about https://github.com/stevedesmond-ca/SimpleGPIO if this gains enough traction and you're looking for speakers!

**Comment by pjgpetecodes** [comment]
> I'm always happy to talk about https://github.com/stevedesmond-ca/SimpleGPIO if this gains enough traction and you're looking for speakers!

Ooo... That looks interesting!
